src/lib/pick-props.js

```diff
@@ -0,0 +1,13 @@
+const isPlainObject = obj =>
+  typeof obj === 'object' && obj !== null && obj.constructor === Object && Object.prototype.toString.call(obj) === '[object Object]'
+
```

Comment:
if we put a lodash function replacement, it seems reasonable if it will have own tests.